config/src/main/java/com/alibaba/nacos/config/server/auth/EmbeddedUserPersistServiceImpl.java

```diff
@@ -39,13 +40,13 @@
 @Conditional(value = ConditionOnEmbeddedStorage.class)
 @Component
 public class EmbeddedUserPersistServiceImpl implements UserPersistService {
-    
+
```

Comment:
Do not change indent please.
please check for all your changed files

Reply:
Github show the indent still be changed

Reply:
I am changing